README.md

```diff
@@ -35,14 +39,38 @@ RESULTS:
         MIN: 5465ms
         MAX: 8540ms
         AVG: 6868ms
+        P50: 5565ms
```

Comment:
I obviously just typed random things that kind of look like they fit (e.g. between MIN MAX)